src/mscorlib/src/System/Diagnostics/Eventing/EventProvider.cs

```diff
@@ -487,7 +475,7 @@ private unsafe void GetSessionInfo(Action<int, long> action)
                 providerInstance = (UnsafeNativeMethods.ManifestEtw.TRACE_PROVIDER_INSTANCE_INFO*)&structBase[providerInstance->NextOffset];
             }
 #else 
-#if !ES_BUILD_PCL && !FEATURE_PAL   // TODO command arguments don't work on PCL builds...
+#if !ES_BUILD_PCL && !FEATURE_PAL  // TODO command arguments don't work on PCL builds...
             // Determine our session from what is in the registry.  
             string regKey = @"\Microsoft\Windows\CurrentVersion\Winevt\Publishers\{" + m_providerId + "}";
             if (System.Runtime.InteropServices.Marshal.SizeOf(typeof(IntPtr)) == 8)
```

Comment:
As mentioned to you privately, I think that change is good, but at the moment, it is more important that we minimize churn between the places where EventSource code is cloned (.NET Native, .NET Core (here), Desktop and the Nuget package).